src/templates/app/_layout/header.jsx

```diff
@@ -196,7 +196,10 @@ const Header = () => (
                                             </div>
                                         </h3>
                                         <div className='account__switcher-list' id='account__switcher-real-list'>
-                                            <a id='account__switcher-add' target='_blank' rel='noopener noreferrer'  className='url-add-account account__switcher-add'>
+                                            <a id='account__switcher-select_currencies' target='_blank' rel='noopener noreferrer' className='url-add-account account__switcher-select_currencies'>
+                                                <span className='account__switcher-select_currencies-text'>{it.L('Select Currency')}</span>
```

Comment:
Same here, should this be in sentence case?

```suggestion
                                                <span className='account__switcher-select_currencies-text'>{it.L('Select currency')}</span>
```